book/03-git-branching/sections/rebasing.asc

```diff
@@ -126,95 +126,73 @@ Ahh, ngunit ang kaligayahan ng pag-rebase ay mayroon ding kakulangan, na maaarin
 
 Kung susundin mo ang patnubay na iyon, magiging maayos ka. Kung hindi, kasusuklaman ka ng mga tao, at kakamuhian ka ng iyong mga kaibigan at pamilya.
 
-When you rebase stuff, you're abandoning existing commits and creating new ones that are similar but different.
-If you push commits somewhere and others pull them down and base work on them, and then you rewrite those commits with `git rebase` and push them up again, your collaborators will have to re-merge their work and things will get messy when you try to pull their work back into yours.
+Kapag ikaw ay nag-rebase ng mga bagay, ikaw ay lumilisan sa umiiral na mga commit at gumagawa ng mga panibago na kahawig ngunit naiiba. Kung magpu-push ka ng mga commit saanman at ang iba ay magpu-pull down ng mga ito at magbabase sa trabaho nito, at pagkatapos ay isusulat mo muli ang mga commit na iyon gamit ang `git rebase` at ipu-push up ang mga ito ulit, ang iyong mga katulong ay kailangang mag merge muli ng kanilang trabaho at ang mga bagay ay magugulo kapag sinubukan mong mag-pull sa kanilang trabaho patungo pabalik sa iyo.
 
-Let's look at an example of how rebasing work that you've made public can cause problems.
-Suppose you clone from a central server and then do some work off that.
-Your commit history looks like this:
+Tumingin tayo sa isang halimbawa kung papaano ang pag-rebase ng trabaho na iyong ginawang publiko ay maaaring magsanhi ng mga problema. Ipagpalagay na ikaw ay nag-clone mula sa isang sentral na server at pagkatapos ay gumawa ng ilang trabaho sa iyon. Ang iyong kasaysayan ng commit ay magmumukhang katulad nito:
 
-.Clone a repository, and base some work on it
-image::images/perils-of-rebasing-1.png["Clone a repository, and base some work on it."]
+.Mag-clone ng isang repositoryo, at magbase ng ilang trabaho nito
+image::images/perils-of-rebasing-1.png["Mag-clone ng isang repositoryo, at magbase ng ilang trabaho nito."]
 
-Now, someone else does more work that includes a merge, and pushes that work to the central server.
-You fetch it and merge the new remote branch into your work, making your history look something like this:
+Ngayon, may iba pang gumagawa ng higit pang trabaho na nagsasama ng isang merge, at nagpu-push ng trabahong iyon sa sentral na server. Nag-fetch ka nito at nag-merge ng bagong remote na branch sa iyong trabaho, ginagawang magkamukha nito ang iyong kasaysayan:
 
-.Fetch more commits, and merge them into your work
-image::images/perils-of-rebasing-2.png["Fetch more commits, and merge them into your work."]
+.Mag-fetch pa ng higit pang mga commit, at i-merge ang mga ito sa iyong trabaho
```

Comment:
Mag-fetch pa ng higit pang mga commit to Kumuha ng maraming mga commit

Reply:
Changing it now.